server/src/controller/trashController.js

```diff
@@ -0,0 +1,58 @@
+import validate from "../util/checkValidation.js";
+import prisma from "../../prisma/client.js";
+import { StatusCodes } from "http-status-codes";
+
+export const createTrash = async (req, res) => {
+    if (validate(req, res)) {
+      return res;
+    }
+    const {weight, categoryId, userId} = req.body;
+    const trashItem = await prisma.trashItem.create({
+        data: {
+            weight,
+            categoryId,
+            userId,
+        },
+    });
+    return res.status(StatusCodes.CREATED).json({ trashItem });
+  };
+
+  export const getTrash = async(req, res) => { 
+    if (validate(req, res)) { //what does this do?
+        return res;
+    }
+    const trash = await prisma.trashItem.findMany({
+        include: {  weight: true, category: true}, //need to talk about what we want to include
```

Comment:
include is for relations so the only things you would include are category and user